libbeat/beat/beat.go

```diff
@@ -107,16 +107,16 @@ type Beat struct {
 	Config    BeatConfig           // Common Beat configuration data.
 	Publisher *publisher.Publisher // Publisher
 
-	filters *filter.Filters // Filters
+	processors *processors.Processors // Processors
 }
 
 // BeatConfig struct contains the basic configuration of every beat
 type BeatConfig struct {
-	Shipper publisher.ShipperConfig   `config:",inline"`
-	Output  map[string]*common.Config `config:"output"`
-	Logging logp.Logging              `config:"logging"`
-	Filters filter.FilterPluginConfig `config:"filters"`
-	Path    paths.Path                `config:"path"`
+	Shipper    publisher.ShipperConfig          `config:",inline"`
+	Output     map[string]*common.Config        `config:"output"`
+	Logging    logp.Logging                     `config:"logging"`
+	Processors processors.ProcessorPluginConfig `config:"processors"`
```

Comment:
The reason why I used `ProcessorPluginConfig` is that it's also used in Metricbeat as you are able to configure `filters` in a Metricbeat module.